cards/spells/chill-touch.tex

```diff
@@ -6,30 +6,19 @@
 
 \CardTraits{Necromancy, Negative}
 
-% TODO: fix formatting to make card fit on one page and remove \footnotesize
-{\footnotesize
 \Flavor{Siphoning negative energy into yourself, your hand radiates a pale darkness.}
 
 Touch 1 living or undead creature.
-}
 
 \Action[Somatic, Verbal]{CRB}{323}{2}{Weaken Living}
 
-\DamageRoll{Negative}{\level{}d4}{spell mod}
+\DamageRoll{Negative}{\level{}d4}{spell mod} (Fortitude Save)
```

Comment:
@Toxaris you explained the difference between _basic Will save_ and _Will save_ to me recently. 

For Chill Touch on living creatures ("Weaken Living") the rules say 

> The target attempts a **basic Fortitude save**, but is also enfeebled 1 for 1 round on a critical failure. 

For Chill Touch on undead creatures the rules say

> The target is flat-footed for 1 round on a failed **Fortitude save**. On a critical failure, the target is also fleeing for 1 round unless it succeeds at a **Will save**.

Thus, should this be
```suggestion
\DamageRoll{Negative}{\level{}d4}{spell mod} (Basic Fortitude Save)
```

Reply:
So my idea was that if we just say "(Fortitude Save)" after some effect, the meaning is:

- if the effect is taking damage, the save results in 2x / 1x / 0.5x / 0x damage (= basic save)
- if the effect is something else, the save results in 1x / 1x / 0x / 0x the effect

in both case, in the paragraph below, we can replace or add save results.

Reply:
Fine with me, but can we make it consistent over the cards? See https://github.com/potsdam-pnp/pf2e-cards/pull/44#pullrequestreview-1608049171